Boccia-Unity/Assets/Boccia/Ramp/BallPresenter.cs

```diff
@@ -268,6 +276,23 @@ private void NavigationChanged()
         }
     }
 
+
```

Comment:
I wonder if this call is expensive in update. 
Would an invisible object to detect collission be better?
That object could just trigger then `BallState == ReadyToRelease`.

Maybe @ekinney-lang could comment on this?